src/utils/update/statement.bo.cease.ts

```diff
@@ -0,0 +1,44 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import { getApplicationData } from "../../utils/application.data";
+import { ApplicationData } from "../../model";
+import { InputDate } from "../../model/data.types.model";
+import { Update } from "../../model/update.type.model";
+import { checkDateValueIsValid } from "../../validation/custom.validation";
+
+const initializeRegistrationDate: InputDate = {
+  day: "31",
+  month: "01",
+  year: "2023"
+};
+
+const updateRegistrationDate = (appData: ApplicationData): ApplicationData => {
+  const date: InputDate = appData.update?.date_of_creation as InputDate;
+  try {
+    checkDateValueIsValid("Registration date does not exist.", date.day, date.month, date.year);
+  } catch (error) {
+    logger.info(error);
+    (appData.update as Update).date_of_creation = initializeRegistrationDate;
+  }
+
+  return appData;
+};
+
+export const getStatmentBOCease = (req: Request, res: Response, next: NextFunction, templateName: string, backLinkUrl: string) => {
```

Comment:
I don't see why this logic can't be done inside the `statement.bo.cease.controller` maybe you it's simpler this way, not sure though

Reply:
this way the controller is much cleaner when more endpoints will be needed to add into controller.